### ISSUE TITLE
Odd behavior with errors.log

### ISSUE THREAD

**Issue by hasufell** [dan-t/hsimport#14]
```
hasufell@localhost ~ $ echo "gaga" > ~/.cache/hsimport/errors.log && hsimport --help
hsimport: gaga
```

I got an error and then every subsequent call to hsimport just `cats` that error forever. What?

**Comment by dan-t** [comment]
Do you have a `hsimport` configuration file (~/.config/hsimport/hsimport.hs)?

> hasufell@localhost ~ $ echo "gaga" > ~/.cache/hsimport/errors.log && hsimport --help
> hsimport: gaga

Why on earth would you do something like this?

`hsimport` uses `dyre` for a Haskell configuration file and `dyre` most
likely just outputs the content of `errors.log`.

**Comment by hasufell** [comment]
> Do you have a `hsimport` configuration file (~/.config/hsimport/hsimport.hs)?

I had, then hsimport threw a ton of build errors, because the public API is broken again. Then I removed my config, but hsimport didn't stop printing tht errors as if it was still recent. I don't think that is expected behavior in any way.

**Comment by dan-t** [comment]
> Then I removed my config, but hsimport didn't stop printing tht
> errors as if it was still recent. I don't think that is expected behavior in
> any way.

Then this is an issue for the `dyre` library and not `hsimport`. I'm not
creating this error file nor caring in any away about it.

**Comment by hasufell** [comment]
I don't know if that is true or if it's because of the way you use the library. I cannot tell and I don't really care either, because I don't use dyre, but hsimport and this issue just shows in hsimport, nowhere else.

**Comment by dan-t** [comment]
I think you misunderstood  what I meant by caring. I don't know anything about this error file and `dyre` doesn't tell me anything about it, but I just get the error as string from `dyre` and outputting it.

`hsimport` can't decide at this point if this is a valid error or not. It's seems to be the job of `dyre` to clean up the error file after a configuration file has been removed.

`hsimport` could now try to delete this error file by itself, but this doesn't seem to be the right
way and might easily break on changes of `dyre`.

I've opened an issue for `dyre` (https://github.com/willdonnelly/dyre/issues/35).
